sockpuppet/consumer.py

```diff
@@ -235,6 +236,14 @@ def render_page(self, reflex):
         reflex_context = {key: getattr(reflex, key) for key in instance_variables}
         reflex_context["stimulus_reflex"] = True
 
+        if not reflex.context._attr_data:
```

Comment:
Does this message show if no context variables are set under either method?

Reply:
The problem with `if len(instance_variables) > 0` is that it would always be true as getmembers actually also picks methods etc. Though it looks like [stackoverflow](https://stackoverflow.com/a/1398059/554903) comes to the rescue ;)